src/gov-action-contracts/nonemergency/SwitchManagerRolesAction.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../../security-council-mgmt/SecurityCouncilManager.sol";
+
+/// @notice Grant the non emergency council the MEMBER_ADDER_ROLE, MEMBER_REPLACER_ROLE, MEMBER_ROTATOR_ROLE and MEMBER_REMOVER_ROLE on the SecurityCouncilManager.
+///         Revoke those same roles from the emergency council.
+contract SwitchManagerRolesAction {
+    SecurityCouncilManager public constant securityCouncilManager =
+        SecurityCouncilManager(0xD509E5f5aEe2A205F554f36E8a7d56094494eDFC);
+
+    address public constant nonEmergencyCouncil = 0xADd68bCb0f66878aB9D37a447C7b9067C5dfa941;
```

Comment:
I think we want the timelock to have the role here, not the SAFE directly

Reply:
we want the safe to have direct control because the manager queues all actions in the L2 timelock, so it goes safe -> manager -> timelock